server/openapi_server/controllers/security_controller.py

```
@@ -0,0 +1,2 @@
+from typing import List
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)